### PR TITLE
Unifying headings and spelling, rephrasing and adding of markdown elements

### PR DESCRIPTION
**Pull request by alexanderdittrich**
## Description
Update to documentation by unifying capitalization of headings and some key words like e.g. "Python", "C++", "MuJoCo", "PAM".
Important information got highlighted using markdown-elements (see "More info: Using real robot").
In a few cases a small rephrasing has been done.
If another site of the documentation has been referenced, a linkage has been added.

## How I Tested
HTML has been generated with Sphinx and inspected on Firefox and Chromium. 

## Remark
I rebased multiple commits and also merged main with the branch to resolve merge conflicts. I double checked that the previous commit history is untouched. However, I would appreciate if the reviewer can give me a small feedback if rebasing and merging was done properly.

### CODE REVIEW

**Review by vincentberenz — APPROVED** [review]
Thanks a lot for the update !

**Comment by alexanderdittrich** [comment]
Just noticed, that there is the same issue with the commit history as in PAM models (where some commits appear twice after rebasing). I will fix this on thursday, before merging.

File: E1_tools.md [hunk 1]
```diff
@@ -52,23 +52,23 @@ To see information about a specific projects:
...
 
 ```bash
 treep --clone PAM_MUJOCO
 ```
 
-### status
+### Status
 
-The status argument is very useful to see the (git) status of each cloned repos (i.e. if they are behind origin, beyond origin, have modified files, etc)
+The status argument is very useful to see the (GIT) status of each cloned repos (i.e. if they are behind origin, beyond origin, have modified files, etc)
```
Comment (luator) [inline E1_tools.md]: Very minor detail: "Git" is not an acronym, so it should be capitalised normally.

**Review by luator — APPROVED** [review]
Looks good to me once the problem with the git history is resolved.

**Pull request merged**